<launch>
   <node pkg="autosail" name="motor_controller_listener" type="motor_controller_node.py">
      <rosparam command="load" file="$(find autosail)/config/motor_controller_config.yaml"/>
   </node>
   <node pkg="autosail" name="deg2rad_rudder" type="deg2rad.py">
      <remap from="/deg" to="/rudder_angle_deg"/>
      <remap from="/rad" to="/rudder_controller/rudder_angle"/>
   </node>
   <node pkg="autosail" name="deg2rad_sail" type="deg2rad.py">
      <remap from="/deg" to="/sail_angle_deg"/>
      <remap from="/rad" to="/sail_controller/sail_servo_angle"/>
   </node>
   <node pkg="autosail" name="rad2deg_rudder" type="rad2deg.py">
      <remap from="/deg" to="/rudder_controller/rudder_angle_deg"/>
      <remap from="/rad" to="/rudder_controller/rudder_angle"/>
   </node>
   <node pkg="autosail" name="rad2deg_sail" type="rad2deg.py">
      <remap from="/deg" to="/sail_controller/sail_servo_angle_deg"/>
      <remap from="/rad" to="/sail_controller/sail_servo_angle"/>
   </node>
</launch>
